admin/upsell.php

```diff
@@ -1,17 +1,17 @@
 <?php namespace HM\BackUpWordPress; ?>
 <div class="hmbkp-upsell">
 
-<p>
-
 <?php
-/** translators:  the 1st placeholder is the first part of the anchor tag with the link to the extensions admin page and the second is the closing anchor tag */
-printf(
-	__( 'Store your backups securely in the Cloud with %1$sour extensions%2$s', 'backupwordpress' ),
-    '<a href="<?php echo esc_url( https://bwp.hmn.md/ ); ?>" target="_blank">',
-    '</a>'
+/** translators:  the 1st placeholder is the first part of the anchor tag with the link to the plugin review page and the second is the closing anchor tag */
+$cta_message = printf(
```

Comment:
That's the best description I heard on it - I went back and forth a few times and decided someone else would have to be able to tell the difference! Thank you again!